qadence_protocols/measurements/protocols.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+import importlib
+from dataclasses import dataclass
+from functools import partial
+
+from qadence import QuantumModel
+from torch import Tensor
+
+from qadence_protocols.protocols import Protocol
+
+PROTOCOL_TO_MODULE = {
+    "tomography": "qadence_protocols.measurements.tomography",
```

Comment:
No shadows ? Ah maybe you plan to add it in another PR.

Reply:
Indeed